index.css

```diff
@@ -0,0 +1,137 @@
+* {
+    box-sizing: border-box;
+    background-size: auto auto;
+    margin: auto auto;
+}
+
+.intro {
+    display: flex;
+    justify-content: space-around;
+    text-align: center;
+    flex-direction: column;
+    background-color: #fafafa;
+    width: 100vw;
+}
+
+.box-container {
+    display: flex;
+    /*height: 300px;*/
+    justify-content: center;
+    /*align-items: center;*/
+    flex-direction: column;
+    background-color: #fafafa;
+    width: 100vw;
+}
+
+.box-1 {
+    width: 50%;
+    max-width: 300px;
+    height: 200px;
+    margin: 20px auto;
+    background-color: #ffffff;
+    border-top: 4px solid #49cfd0;
+    border-radius: 5px;
+    box-shadow: 0 4px 6px 0 rgba(34, 60, 80, .16);
+    transition: all ease-in-out .2s;
```

Comment:
I don't know this transition, ease-in-out. What it does?

Reply:
I used this in the little basic website I built a month before the boot camp following online tutorial. It's about the animation parameters but i actually don't think i've done it correctly in here.

On my other website there was another bit of code for profile picture (hover) so that when you moved your mouse over the profile picture, the box shadow changed slightly and you could see that change for .2 seconds.

I'll send you the links with the info about it in Slack.